### PR TITLE
Add sendemail=1 to order_carriers endpoint

### PR DESCRIPTION
**Pull request by jf-viguier**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Add sendemail=1 to order_carriers endpoint
| Fixed ticket? | N/A

### CODE REVIEW

**Review by PrestaEdit — APPROVED** [review]

File: webservice/getting-started.md [hunk 1]
```diff
@@ -234,6 +234,8 @@ The `language=1` or `language=[1|2]` parameter can be used to return only these
 
 The `sendemail=1` parameter can be used if you need to change the state of an order AND you want the emails to be sent to the customer: you will have to do a **POST** on `http://example.com/api/order_histories?sendemail=1`
 
+The `sendemail=1` parameter can be used on order_carriers endpoint to send the in transit email with the tracking number. Example : `http://example.com/api/order_carriers/12345?sendemail=1`
```
Comment (kpodemski) [inline webservice/getting-started.md]:
```suggestion
The `sendemail=1` parameter can be used on the `order_carriers` endpoint to send the _in-transit_ email with the tracking number. Example: `http://example.com/api/order_carriers/12345?sendemail=1`
```

Could you be more precise and describe what `12345` stands for?

**Review by kpodemski — CHANGES_REQUESTED** [review]
Feedback

**Comment by jf-viguier** [comment]
@kpodemski it's done

**Comment by jf-viguier** [comment]
@kpodemski ?

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks, @jf-viguier, and sorry for the long waiting. A lot to do.

I guess it should be backported to 1.7.x? Let me know.

**Pull request merged**